LIGHTNING-API-BLUEPRINT.apib

```diff
@@ -21,13 +21,18 @@ Clients should ignore unknown parameters, to avoid technical issues.
 
 # Group Lightning
 
-## Retrieve lightning observation [?provider={provider1,provider2}&locationWithin={lonTopLeft,latTopLeft,lonBottomRigth,latBottomRigth}&occurrenceFrom={startTime}&occurrenceUntil={endTime}]
-## Retrieve lightning observation [?provider={provider1,provider2}&locationWithin={lonTopLeft,latTopLeft,lonBottomRigth,latBottomRigth}&occuredFrom={startTime}]
+## Retrieve lightning observation [?provider={provider1,provider2}&locationWithin={lonTopLeft,latTopLeft,lonBottomRight,latBottomRight}&occurrenceFrom={startTime}&occurrenceUntil={endTime}]
+## Retrieve lightning observation [?provider={provider1,provider2}&locationWithin={lonTopLeft,latTopLeft,lonBottomRight,latBottomRight}&occurredFrom={startTime}]
```

Comment:
First line contains occurrenceFrom and second occurredFrom. What name is correct?